src/app/components/TxDetailsEdit.tsx

```diff
@@ -0,0 +1,88 @@
+import { formatEther, parseUnits } from '@ethersproject/units';
+import { Body } from '@mycrypto/ui';
+import type { ComponentProps } from 'react';
+import type { FormState } from 'typed-react-form';
+
+import { translateRaw } from '@common/translate';
+import { getChain } from '@data';
+import type { TransactionRequest } from '@types';
+import { bigify } from '@utils';
+
+import { Box } from '.';
+import { FormInput } from './FormInput';
+import { FormTextArea } from './FormTextArea';
+import { TxDetailsBlockRow as BlockRow } from './TxDetailsBlockRow';
+import { TxDetailsRow as Row } from './TxDetailsRow';
+
+const InputRow = ({
+  label,
+  unit,
+  ...props
+}: { label: string; unit?: string } & ComponentProps<typeof FormInput>) => (
+  <Row
+    label={label}
+    value={
+      <Box variant="horizontal-start" width="45%">
+        {/* @ts-ignore for now */}
```

Comment:
Why?

Reply:
Still having type errors even when doing that 😢

Reply:
Yes, but those should be easier to fix.

Reply:
Finally fixed it 😆